evalml/model_understanding/__init__.py

```diff
@@ -12,6 +12,7 @@
     graph_binary_objective_vs_threshold,
     partial_dependence,
     graph_partial_dependence,
-    graph_prediction_vs_actual
+    graph_prediction_vs_actual,
+    graph_prediction_vs_target_over_time
```

Comment:
@freddyaboulton do you like "graph_prediction_vs_actual" or "graph_prediction_vs_target" more? We should use the same name for both of these.

I don't feel strongly about it, but I am more used to "prediction vs actual"

Reply:
Now that I've seen predicted vs actual, I agree that that sounds much better :o

Reply:
Sounds good! I preferred using "target" in the function name since that's what we mention in the docstrings and axis labels but I'll change the name to actual for consistency.